neptune_fastai/impl/__init__.py

```diff
@@ -207,11 +207,14 @@ def _total_model_parameters(self) -> int:
 
     @property
     def _trainable_model_parameters(self) -> int:
-        return sum([p.numel() for p in trainable_params(self.learn)])
+        return sum(p.numel() for p in trainable_params(self.learn))
 
     @property
     def _optimizer_criterion(self) -> str:
-        return repr(self.loss_func.func)
+        if hasattr(self.learn.loss_func, 'func'):
+            return repr(self.loss_func.func)
+        else:
+            return repr(self.loss_func)
```

Comment:
Ok, I see that it will be prefixed with `FlattenedLoss of ` -> https://github.com/fastai/fastai/blob/0a01eba7de66bd5430295d09517a0ad530d9ff66/fastai/losses.py#L34

But I see a non-consequence in attribute access as in If statement there is `self.learn.loss_func` vs "direct" `self.loss_func` below.

Reply:
And I would recommend a more compact way:
```python
return repr(self.loss_func.func if hasattr(self.loss_func, 'func') else self.loss_func)
```

Reply:
Custom losses don't have the attr`func`